docs.gs

```diff
@@ -0,0 +1,17 @@
+function onOpen() {
+  const menu = DocumentApp.getUi().createMenu('Custom Menu')
+  menu.addItem('Add 1', 'this.addOne')
+  menu.addToUi()
+}
+
+function addOne(){
+  const sel = DocumentApp.getActiveDocument().getSelection().getRangeElements()
+  for (let i = 0; i < sel.length; i++){
+    let tab = sel[i].getElement().editAsText()
+    if (tab.getText() != ''){
+      let num = Number(tab.getText())
```

Comment:
I didn't know that there was a tab.getText function. Can you tell me more about that?